frontend/public/index.html

```diff
@@ -4,6 +4,12 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>IO Team B</title>
+    <link
```

Comment:
dalej nie rozumiem po co jest nam potrzebny ten plik

Reply:
[https://stackoverflow.com/questions/42438171/wheres-the-connection-between-index-html-and-index-js-in-a-create-react-app-app](url)

Reply:
[https://stackoverflow.com/questions/41738421/how-react-js-index-js-file-contacting-index-html-for-id-references](url)

Reply:
https://stackoverflow.com/questions/41738421/how-react-js-index-js-file-contacting-index-html-for-id-references